wgpu-core/Cargo.toml

```diff
@@ -57,7 +57,7 @@ version = "0.12"
 hal = { path = "../wgpu-hal", package = "wgpu-hal", version = "0.12", features = ["gles"] }
 
 [target.'cfg(all(not(target_arch = "wasm32"), any(target_os = "ios", target_os = "macos")))'.dependencies]
-hal = { path = "../wgpu-hal", package = "wgpu-hal", version = "0.12", features = ["metal"] }
+hal = { path = "../wgpu-hal", package = "wgpu-hal", version = "0.12", features = ["vulkan", "metal"] }
```

Comment:
I don't think we'd want to do this unconditionally. Vulkan Portability path isn't conventional and shouldn't encumber the builds on all the Apple platforms. It should be behind a feature instead.

Reply:
@kvark Following your guide, add a `portability` feature to support vulkan